checker/check_result.go

```diff
@@ -56,17 +56,14 @@ const (
 var errSuccessTotal = errors.New("unexpected number of success is higher than total")
 
 // CheckResult captures result from a check run.
-//
-//nolint:govet
 type CheckResult struct {
-	Name    string
-	Version int
```

Comment:
similar comment on human-readability. We should decide if we care about this

Reply:
> So it helps to have performance IMO. Thoughts?

My first thought is, does it matter? I know we have the linter (and could disable it if the answer is no), but is this a performance bottleneck? Compared to the I/O and API operations we need to wait for anyway?

Reply:
The compiler utilizes the L1 cache based on the word size.

https://articles.wesionary.team/simple-data-alignment-technique-to-speed-up-your-struct-in-golang-954c43faa1fe
https://www.ardanlabs.com/blog/2013/07/understanding-type-in-go.html

It helps to make it better. 

> I understand API Operations.